pkg/kubelet/prober/prober.go

```diff
@@ -277,3 +285,30 @@ func (eic execInContainer) SetStderr(out io.Writer) {
 func (eic execInContainer) Stop() {
 	//unimplemented
 }
+
+// selectProbeIP selects a pod IP for probes from the specified PodStatus
+// based on the specified preferredIPFamily
```

Comment:
Document behavior - this selects the first record of a given family.

Is that what we want?  Or would we be better off designating the IP as "use this for probes" via the `status.PodIPs` structs themselves?